code/game/machinery/cryo.dm

```diff
@@ -601,26 +601,10 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	set name = "Eject occupant"
 	set category = "Object"
 	set src in oview(1)
-	if(panel_open)
-		to_chat(usr, "<span class='bnotice'>Close the maintenance panel first.</span>")
-		return
-	if(usr == occupant)//If the user is inside the tube...
-		if (usr.isDead())//and he's not dead....
-			return
-		to_chat(usr, "<span class='notice'>Release sequence activated. This will take thirty seconds.</span>")
-		sleep(300)
-		if(!src || !usr || !occupant || (occupant != usr)) //Check if someone's released/replaced/bombed him already
-			return
-		go_out()//and release him from the eternal prison.
-	else
-		if (usr.isUnconscious() || istype(usr, /mob/living/simple_animal))
-			return
-		go_out()
-	add_fingerprint(usr)
-	return
+	AltClick(usr)
```

Comment:
Don't really know if `user` would work here.

Reply:
it will if you define it as argument for the proc